src/fairing.rs

```diff
@@ -122,19 +122,60 @@ impl Fairing for Compression {
 ///
 /// rocket::build()
 ///     // ...
-///     .attach(CachedCompression::fairing(vec![".otf", "main.dart.js"]))
+///     .attach(CachedCompression {
+///       cached_paths: vec!["", "/", "/about", "/people", "/posts", "/events", "/groups"],
+///       cached_path_prefixes: vec!["/user/", "/g/", "/p/"],
+///       cached_path_suffixes: vec![".otf", "main.dart.js"]
+///     })
 ///     // ...
 ///     # ;
 ///
 /// ```
 pub struct CachedCompression {
-    pub cached_path_endings: Vec<&'static str>,
+    pub cached_paths: Vec<&'static str>,
```

Comment:
With this more complicated configuration scheme, I think it would be useful to convert these from `&'static str` to `String`.

I think that will provide users with more flexibility if they need to do things like configure paths/prefixes/suffixes dynamically from something like config file, environment variable, etc.